conf/cassandra-env.sh

```diff
@@ -236,6 +236,8 @@ else
   JVM_OPTS="$JVM_OPTS -Dcom.sun.management.jmxremote.authenticate=true"
 
   # jmx ssl options
+  # Consider using the jmx_encryption_options section of cassandra.yaml instead to prevent sensitive information being
```

Comment:
If both `cassandra.yaml` and `cassandra-env.sh` are configured the precedence is cassandra-env.sh since that is the prior functionality before this change. You can see this condition check [here](https://github.com/apache/cassandra/pull/3638/files#diff-039489ddb488470626dad5fac18d1541da0bd23f3137178cf3f63b29e020d5daR55) in the code. However, this means we should clarify this in the documentation update as well as in the cassandra-env.sh?